mm/2s2h/Enhancements/Cutscenes/MiscInteractions/SkipMoonCrash.cpp

```diff
@@ -0,0 +1,35 @@
+#include <libultraship/bridge.h>
+#include <libultraship/luslog.h>
```

Comment:
Change the bridge import and remove the unused luslog one.
```suggestion
#include "public/bridge/consolevariablebridge.h"
```